### DISCUSSION TITLE
Option to hide the caret before the first letter has been typed

### DISCUSSION THREAD

**Discussion by Banzobotic** [monkeytypegame/monkeytype#1310] [Ideas]
Sometimes I get the first letter wrong because it was hard to read to first letter due to it being partially covered by the caret. Having a setting that hides the caret until you start typing would solve this issue.

**Comment by Miodec** [comment]
I feel like adding a setting for something so specific is overkill in my opinion. Besides, thats why the caret is flashing before you press the first key. Maybe the flashing animation could be adjusted slightly
